src/components/app/ListOfPublishedProfiles.js

```diff
@@ -135,25 +136,31 @@ class PublishedProfile extends React.PureComponent<
           publishedProfilesListItem_ConfirmDialogIsOpen: confirmDialogIsOpen,
         })}
       >
-        <a
-          className="publishedProfilesLink"
-          href={urlPath}
-          title={`Click here to load profile ${smallProfileName}`}
+        <Localized
+          id="ListOfPublishedProfiles--publishedProfilesLink"
+          attrs={{ title: true }}
+          vars={{ smallProfileName: smallProfileName }}
         >
-          <div className="publishedProfilesDate">
-            {_formatDate(
-              uploadedProfileInformation.publishedDate,
-              nowTimestamp
-            )}
-          </div>
-          <div className="publishedProfilesInfo">
-            <div className="publishedProfilesName">
-              <strong>{profileName}</strong> (
-              {_formatRange(uploadedProfileInformation.publishedRange)})
+          <a
+            className="publishedProfilesLink"
+            href={urlPath}
+            title={`Click here to load profile ${smallProfileName}`}
+          >
```

Comment:
@canova added this recently. Review please.

Reply:
Thanks, this looks good